lib/src/rewrite.rs

```diff
@@ -462,6 +462,9 @@ pub struct MoveCommitsStats {
     pub num_skipped_rebases: u32,
 }
 
+/// Rebase and move `target_commits` in graph based on new parent and child
+/// nodes
+///
 /// Moves `target_commits` from their current location to a new location in the
 /// graph, given by the set of `new_parent_ids` and `new_children`.
```

Comment:
nit: I think the first paragraph repeats the same thing. Maybe we can just insert blank line here?